morpho5/docs/color.md

```diff
@@ -14,11 +14,69 @@ Create a Color object from an RGB pair:
 
     var col = Color(0.5,0.5,0.5) // A 50% gray
 
+The `color` module also provides `ColorMap`s, which are give a sequence of colors as a function of a parameter; these are useful for plotting the values of a `Field` for example.
+
 [showsubtopics]: # (subtopics)
 
+## RGB
+[tagrgb]: # (rgb)
+
+Gets the rgb components of a `Color` or `ColorMap` object as a list. Takes a single argument in the range 0..1, although the result will only depend on this argument if the object is a `ColorMap`.
```

Comment:
Minor point, but maybe "in the range 0 to 1" would be better since "0..1" has a specific meaning in Morpho and just means 0 and 1.

Reply:
Fixed!